src/demo-app/app/github-issues/issue.5345.demo.ts

```diff
@@ -1,28 +1,36 @@
-import {Component, ViewEncapsulation} from '@angular/core';
+import {Component} from '@angular/core';
 
 @Component({
+  moduleId: module.id,
   selector: 'demo-issue-5345',
-  styleUrls: [
-    'issue.5345.demo.css',
-    '../demo-app/material2.css'
-  ],
+  styleUrls: [ 'issue.5345.demo.css' ],
```

Comment:
No spaces inside the braces

(this seems more prevalent, probably better for follow-up PR)